### DISCUSSION TITLE
Initial Release

### DISCUSSION THREAD

**Discussion by phongthanhbuiit** [phongthanhbuiit/whisper-realtime-gui#1] [Announcements]
# Speech To Text v1.0.0 - Initial Release

We're excited to announce the first release of our Speech To Text application! This modern, real-time speech recognition application leverages OpenAI's Whisper model to provide accurate transcription with a beautiful native interface.

## 🌟 Key Features

- Real-time audio transcription using OpenAI's Whisper
- Beautiful, modern UI with animated audio visualizer
- GPU acceleration support (Apple Silicon/CUDA)
- Multi-language support
- Live audio waveform visualization with dynamic effects
- Multiple Whisper model options (tiny, base, small, medium, large)
- Optimized streaming for better real-time performance

## 💻 Installation

1. Download the Speech-To-Text.dmg file
2. Open the downloaded .dmg file
3. Drag the application to your Applications folder
4. Double click to run the application

## 🔧 System Requirements

- macOS (optimized for Apple Silicon)
- 4GB RAM minimum (8GB recommended)
- GPU recommended for better performance

## 🐛 Known Issues

None reported yet. If you encounter any issues, please report them in the Issues section.

## 📝 Notes

This is our initial release focusing on macOS support. Future updates will bring additional features and improvements based on user feedback.

## 🙏 Acknowledgments

Special thanks to OpenAI for the Whisper model that powers our transcription engine.

<hr /><em>This discussion was created from the release <a href='https://github.com/phongthanhbuiit/whisper-realtime-gui/releases/tag/'>Initial Release</a>.</em>

**Comment by dzg-dot** [comment]
Em ko có kiến thức về lập trình, xin hỏi ad có thể ra bản sử dụng được trên điện thoại được không ạ?

**Reply by phongthanhbuiit** [reply]
hiện tại chưa có model AI Speech to text mà chạy được trên điện thoại nên anh không tạo ra được. Em có thể dùng chatgpt hay nhiều phần mềm khác nhé do các phần mềm đó người ta chạy model AI trên server chứ không phải phần cứng vật lý.
